source/perses/perses.go

```diff
@@ -90,10 +90,18 @@ func (c *persesCollector) generateUsage(metricNames modelAPIV1.Set[string], curr
 	dashboardURL := fmt.Sprintf("%s/api/v1/projects/%s/dashboards/%s", c.persesURL, currentDashboard.Metadata.Project, currentDashboard.Metadata.Name)
 	for metricName := range metricNames {
 		if usage, ok := metricUsage[metricName]; ok {
-			usage.Dashboards.Add(dashboardURL)
+			usage.Dashboards.Add(modelAPIV1.DashboardUsage{
+				UID:  currentDashboard.Metadata.Name,
```

Comment:
probably here if you want an accurate ID, it will be `currentDashboard.Metadata.Project/currentDashboard.Metadata.Name`